front/app/modules/commercial/insights/admin/components/TopBar/index.tsx

```diff
@@ -129,9 +129,9 @@ const TopBar = ({
               <DropdownListItem onClick={handleDeleteClick}>
                 {formatMessage(messages.delete)}
               </DropdownListItem>
-              <DropdownListItem>
+              {/* <DropdownListItem>
```

Comment:
I've commented out the duplicate buttons because we don't have it implemented yet.